consts.go

```diff
@@ -3,10 +3,15 @@ package bugsnagperformance
 import "time"
 
 const (
-	samplingAttribute      = "bugsnag.sampling.p"
-	samplingResponseHeader = "Bugsnag-Sampling-Probability"
-	samplingRequestHeader  = "Bugsnag-Span-Sampling"
-	fetcherRetryInterval   = 30 * time.Second
-	fetcherRefreshInterval = 24 * time.Hour
-	fetcherRequestBody     = `{"resourceSpans": []}`
+	samplingAttribute       = "bugsnag.sampling.p"
+	samplingResponseHeader  = "Bugsnag-Sampling-Probability"
+	samplingRequestHeader   = "Bugsnag-Span-Sampling"
+	fetcherRetryInterval    = 30 * time.Second
+	fetcherRefreshInterval  = 24 * time.Hour
+	fetcherRequestBody      = `{"resourceSpans": []}`
+	deploymentEnvAttribute  = "deployment.environment"
+	serviceVersionAttribute = "service.version"
+	bugsnagTelemetrySDKName = "bugsnag.telemetry.sdk.name"
+	bugsnagTelemetrySDKVer  = "bugsnag.telemetry.sdk.version"
```

Comment:
nit: consistency with existing const names

```suggestion
	bugsnagSDKNameAttribute = "bugsnag.telemetry.sdk.name"
	bugsnagSDKVersionAttribute  = "bugsnag.telemetry.sdk.version"
```